src/sentry_plugins/amazon_sqs/plugin.py

```diff
@@ -132,20 +131,27 @@ def forward_event(self, event, payload):
                 message["MessageDeduplicationId"] = uuid4().hex
             client.send_message(**message)
         except ClientError as e:
-            if six.text_type(e).startswith("An error occurred (AccessDenied)"):
+            if six.text_type(e).startswith(
+                "An error occurred (InvalidClientTokenId)"
+            ) or six.text_type(e).startswith("An error occurred (AccessDenied)"):
                 # If there's an issue with the user's token then we can't do
                 # anything to recover. Just log and continue.
                 metrics_name = "sentry_plugins.amazon_sqs.access_token_invalid"
                 logger.info(
                     metrics_name,
                     extra={
-                        "queue_url": queue_url,
-                        "access_key": access_key,
-                        "region": region,
+                        "test_number": 1,
                         "project_id": event.project.id,
                         "organization_id": event.project.organization_id,
                     },
                 )
+                logger.info(
+                    metrics_name,
+                    extra={"test_number": 2, "organization_id": event.project.organization_id},
+                )
+                logger.info(
+                    metrics_name, extra={"test_number": 3, "project_id": event.project.id},
```

Comment:
@ceorourke  I would avoid adding this `test_number` in extra in case that causes issues. We can figure out what test it was based on what info was there. Also, you should definitely have another one where you omit `extra` altogether.